src/Client.php

```diff
@@ -261,43 +262,40 @@ public function userReport(string $report_uid,
     /**
      * {@inheritdoc}
      */
-    public function userReportMake(string $report_type_uid,
-                                   string $type,
-                                   string $value,
-                                   ?array $options = [],
-                                   ?bool $is_force = false,
-                                   ?string $on_update = null,
-                                   ?string $on_complete = null,
-                                   ?array $data = null): UserReportMakeResponse
+    public function userReportMake(ReportMakeParams $params): UserReportMakeResponse
     {
-        $request_options = [];
+        $request_body = [
+            'queryType' => $params->getType(),
+            'query'     => $params->getValue(),
+        ];
 
-        if (\is_bool($is_force)) {
-            $request_options['FORCE'] = $is_force;
-        }
+        $options = [
+            'FORCE' => $params->isForce(),
+        ];
 
-        if (\is_string($on_update)) {
-            $request_options['webhook']['on_update'] = $on_update;
+        if ($params->getOnUpdateUrl() !== null) {
+            $options['webhook']['on_update'] = $params->getOnUpdateUrl();
```

Comment:
```suggestion
        if (\is_string($on_update = $params->getOnUpdateUrl())) {
            $options['webhook']['on_update'] = $on_update;
```

I think is seems more strict. And you?

>>>
Here, and in all similar conditions in this method.
>>>

Reply:
Yes, it looks good! Fixed 0f9be3a9ff8c45be956fd266ff31303e603c1c14